spec/models/tag_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe Tag do
+
+  it "decreases tag_count when a debate is hidden" do
+    debate = create(:debate)
+    tag = create(:tag)
```

Comment:
why not creating the variables in `let`? It is something I generally like as then you find all the variables definitions together and always use lazy evaluation, but in the case of `tag` would also avoid having repeated code, as this variables is created twice, also for the second test. :confounded: